CMakeLists.txt

```diff
@@ -108,15 +108,14 @@ if(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "arm")
 	target_link_directories(MainLoopLib PUBLIC "$ENV{LIBI2C_DIRECTORY}/lib")
 	
 else()
+	target_link_libraries(MainLoopLib systemd)
```

Comment:
Heads up that this is another dynamically linked library, so no support for cross-compile